lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -732,57 +733,98 @@ def _get_default_parameters(kymo, channel):
+            "How big a particle needs to appear to be tracked as a single molecule.",
             "float",
-            2 * position_scale,
-            *(1.0 * position_scale, 5.0 * position_scale),
+            4 * position_scale,
+            *(0.0, 15.0 * position_scale),
             True,
+            r"Minimum particle width defines how big a particle needs to appear as on the "
+            r"kymograph for it to be tracked as a single molecule. This parameter should be set to "
+            r"roughly the width of the point spread function. Making it larger rejects more noise, "
+            r"but at the cost of potentially merging tracks that are close together.",
         ),
         "vel": KymotrackerParameter(
-            "Velocity",
+            "Expected velocity",
             "How fast does the particle move?",
             "float",
             0.0,
             *(-5.0 * vel_calibration, 5.0 * vel_calibration),
             True,
+            "Minimum particle velocity defines how fast particles can move on along the DNA on "
+            "average. When tracking, the algorithm searches for points in future scan lines to "
+            "connect. Points within a certain distance from the expected future position are "
+            "connected.",
         ),
         "diffusion": KymotrackerParameter(
-            "Diffusion", "Expected diffusion constant.", "float", 0.0, *(None, None), False
+            "Diffusion",
+            "Expected diffusion constant.",
+            "float",
+            0.0,
+            *(None, None),
+            False,
+            "When tracking, the algorithm searches for peaks in future frames to connect. Peaks "
+            "within a certain distance from the expected future position are connected. The "
+            "diffusion parameter determines how quickly this distance grows over time.",
         ),
         "sigma_cutoff": KymotrackerParameter(
             "Sigma cutoff",
-            "Number of standard deviations from the expected trajectory a particle no longer belongs to a trace.",
+            "Number of standard deviations from the expected trajectory a particle no longer "
+            "belongs to a trace.",
             "float",
             2.0,
             *(None, None),
             False,
-        ),
-        "min_length": KymotrackerParameter(
-            "Min length",
-            "Minimum number of frames a spot has to be detected in to be considered.",
-            "int",
-            3,
-            *(1, 10),
-            True,
+            "Search range controls how much the location can fluctuate from one time point to the "
+            "next while still being considered part of the same track. Sigma cutoff controls the "
+            "actual threshold applied to the expected positional variability. Larger values for "
+            "sigma or sigma_cutoff will be more permissive when it comes to connecting detected "
+            "peaks into a track.",
```

Comment:
Again, not shown currently, but considering at the moment we're still referring to this one with "sigma" but not the other, I think the description should make the connection between the 2 parameters that are related

```suggestion
            "Search range (sigma) controls how much the location can fluctuate from one time point to the "
            "next while still being considered part of the same track. Sigma cutoff controls the "
            "actual threshold applied to the expected positional variability. Larger values for "
            "sigma or sigma_cutoff will be more permissive when it comes to connecting detected "
            "peaks into a track.",
```

Reply:
👍